Core/PixelExperiment.swift

```diff
@@ -97,15 +98,8 @@ final internal class PixelExperimentLogic {
             return cohort
         }
 
-        let randomNumber = Int.random(in: 0..<100)
-
-        // Allocate user to a cohort based on the random number
-        let cohort: PixelExperiment
-        if randomNumber < 50 {
-            cohort = .control
-        } else {
-            cohort = .newSettings
-        }
+        let variants: [PixelExperiment] = [.control, .breakageSiteReportingFlowA, .breakageSiteReportingFlowB]
+        let cohort: PixelExperiment = variants[Int.random(in: 0..<3)]
```

Comment:
Not a dealbreaker but you can do:
```suggestion
        let cohort: PixelExperiment = variants[Int.random(in: array.indices)]
```
and have safe indexing regardless of item count (unless the collection is empty).